src/service/video-manager-impl.js

```diff
@@ -350,11 +350,30 @@ export class VideoManager {
         return this.entries_[i];
       }
     }
+
     dev().assert(false, 'video is not registered to this video manager');
     return null;
   }
 
   /**
+   * Get the current analytics details for the given video.
+   * Silently fail if the video is not found in this manager.
+   * @param {!AmpElement} videoElement
+   * @return {!Promise<!../video-interface.VideoAnalyticsDetailsDef>|!Promise<undefined>}
+   */
+  getVideoAnalyticsDetails(videoElement) {
+    for (let i = 0; i < this.entries_.length; i++) {
```

Comment:
`getEntryForVideo_` takes the video instance not the element. Maybe we should add a `getEntryForElement_` method :)